test/integration/tests.js

```diff
@@ -65,6 +65,21 @@ test('Gets content types', async (t) => {
   t.ok(response.items, 'items')
 })
 
+test('Gets a content type that has resource links', async (t) => {
+  t.plan(9)
```

Comment:
whats `plan` for?

Reply:
I figured out that `plan` is for the number assertions that will follow. All other tests in this file were using it. That's the reason I used it as well.